src/node_lttng_provider.h

```diff
@@ -0,0 +1,100 @@
+#ifndef SRC_NODE_LTTNG_PROVIDER_H_
+#define SRC_NODE_LTTNG_PROVIDER_H_
+
+#define TRACEPOINT_CREATE_PROBES
+#define TRACEPOINT_DEFINE
+#include "node_lttng_tp.h"
+
+namespace node {
+
+void NODE_HTTP_SERVER_REQUEST(node_lttng_http_server_request_t* req,
+    node_lttng_connection_t* conn, const char *remote, int port,
+    const char *method, const char *url, int fd) {
+  tracepoint(node, http_server_request,
+    req->url, req->method, req->forwardedFor);
```

Comment:
Style: arguments that continue on the next line should align with the arguments on the line above (goes for the function definition a few lines up and the function call here.)